docs/Metrics.md

```diff
@@ -0,0 +1,276 @@
+      `grep ^coturn_ /tmp/report.om |wc -lc`. The first number in the output
+      is the number of lines and thus number of metrics in the report, the
+      2nd number is the size of the report without any `# TYPE` or `# HELP`
+      comments.
+    - to find out, how many bytes you can drop from transmission of each report
+      using the option **--prom-compact**, one may run:
+      `grep -v ^coturn_ /tmp/report.om |wc -lc`.
+
+- you may also try the same test using an invalid password or running a test
+  peer e.g. on machineC (`turnutils_peer -v -p 16384`) and add the options
+  `-e machineC -r 16384`to the `turnutils_uclient` command.
+
+
+# Misc options
+
+## **--prom-compact**
+Should be used on all production systems to avoid the inclusion of the `HELP`
+and `TYPE` comments for each metric in each report.
+Usually it is sufficient to get and record them once, when a demo is required
+or Grafana charts get setup. There is no need to send them again and again in
+each report - would just waste bandwidth and demands more resources to parse
+and process the report. At least on production systems it is recommended to
+enable this option.
+
+## **--prom-realm**
+Label the `session_state` metric with the session realm. If one uses different
+realms for user authentication it could make sense to add a realm label, so one
+may select the interested time series by the realm. If you only use one and the
+same realm it is usually just overhead and should be skipped. Because rarely
+used, it is not taken into account in the calculations of number of metrics
+above.
+
+## **--prom-usernames**
+Label the `session_state` metric with client usernames. One should not do this,
+unless really needed: Modern apps often generate usernames on the fly (e.g.
+hash(currentTimeAsString + `:` + username)) and thus all the names are
+different or even unique for a long time. This in turn means, that a new state
+metric gets created for each new session. This stresses the server resources
+the same way as `--prom-uniq-sid` does, because a sample for each metric gets
+kept in memory and needs to be formatted and transferred on each /metric
+request. When the report gets collected and stored in a time series DB, this
+would in turn create a new time series for each new session
+(metricName + label:value pairs form the key for a time series) and would let
+your DB "explode", get it performance and resource wise probably into trouble,
+because it has to dig into such a huge set of time series (high cardinality)
+and replace "active/cached" ones (high churn rate). So do NOT use it, unless
+you known what you are doing!
+
+
+# More Information
+More information related to labels and cardinality can be found e.g. via:
+- https://docs.victoriametrics.com/keyConcepts.html
+- https://docs.victoriametrics.com/FAQ.html#what-is-high-cardinality
+- https://docs.victoriametrics.com/FAQ.html#what-is-high-churn-rate
+
```

Comment:
Add in here:
# Usage
  E.g. add the grafana dashboard configure and usage documents.
# [Metrics setup](Prometheus.md)